components/style/values/animated/svg.rs

```diff
@@ -0,0 +1,92 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+//! Animated types for CSS values in SVG
+
+use app_units::Au;
+use values::animated::{Animate, Procedure};
+use values::computed::{NonNegativeNumber, Number, Percentage};
+use values::computed::length::{CalcLengthOrPercentage, LengthOrPercentage};
+use values::computed::length::NonNegativeLengthOrPercentage;
+
+/// Stroke-* value support unit less value, so servo interpolate length value as number.
+#[derive(Clone, ComputeSquaredDistance, Copy, Debug, PartialEq, ToAnimatedZero)]
+#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+#[allow(missing_docs)]
+pub enum SvgNumberOrPercentageOrCalc {
```

Comment:
I still don't get it, why isn't that named `SvgLengthOrPercentageOrNumber` just as the non-animated version?